CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 cmake_minimum_required(VERSION 3.15)
 include (ExternalProject)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
what functionality do we need from C++14?

Reply:
Conclusion: We decided to go with c++14